### ISSUE TITLE
results filtering UX on the results page

### ISSUE THREAD

**Issue by berger-n** [buda-base/public-digital-library#880]
currently we have a slightly different set of widgets depending on the results type (Person, Topic, Work etc.), and only a few of them are open by default, when they aren't the user has to click the title to display the checkboxes:

- [Versions](https://library.bdrc.io/search?q=%22spyod%20%27jug%22~1&lg=bo-x-ewts&t=Instance&pg=1):

![image](https://github.com/buda-base/public-digital-library/assets/33383857/18dd8ae5-8dda-4c47-9329-a381e55cc653)

- [Persons](https://library.bdrc.io/search?q=%22spyod%20%27jug%22~1&lg=bo-x-ewts&t=Person):

![image](https://github.com/buda-base/public-digital-library/assets/33383857/8b7392bb-a3b5-4a8d-b4c9-ba8de671ad36)

---

should something like this be implemented on the new results page? 

and something related, should the empty widgets titles like *Translator* be removed in that case?
  https://library-dev.bdrc.io/osearch/search?q=spyod%20%27jug&associatedCentury%5B0%5D=19

![image](https://github.com/buda-base/public-digital-library/assets/33383857/e8a0f1a3-313e-4ad5-936d-fc200592117c)

**Comment by roopeux** [comment]
Empty widgets should be removed.

I think the widgets do add value, but I assume they will not play a very big role in the new UX.  Once we see some real users on this UI we will know much more.

But since we don't limit the scope to "person" etc. anymore, the widgets should be whatever the "aggs" query returns from Opensearch, I guess. @eroux

**Comment by eroux** [comment]
yes, the current aggs work well, I guess one question is do we use the older design for the "type" opensearch agg?

![343697321-18dd8ae5-8dda-4c47-9329-a381e55cc653](https://github.com/buda-base/public-digital-library/assets/60868/e4100827-c04b-42e6-9016-6cef703a84be)

I don't have a strong opinion either way, it's just that keeping a similar design is a smaller change
